test/libsolidity/util/TestFunctionCall.h

```diff
@@ -44,18 +44,24 @@ class TestFunctionCall
 public:
 	enum class RenderMode
 	{
-		ExpectedValuesExpectedGas,
-		ActualValuesExpectedGas,
-		ExpectedValuesActualGas
+		ExpectedValues,
+		ActualValues
+	};
+	enum class GasRenderMode
+	{
+		ExpectedGas,
+		ActualGas
 	};
```

Comment:
Maybe this change should go into a separate PR? It's not strongly related to this PR (i.e. you can just ignore the gas and treat `ExpectedValuesExpectedGas` and `ExpectedValuesActualGas` as both meaning "expected values") and the problem is that it enables the `ActualValuesActualGas` case that was not possible before. I'm not certain that after your changes that case is handled correctly and maybe it would be easier to deal with it in isolation.

Reply:
I was thinking exactly the same, these changes made it harder to read this PR. I removed them and we could probably create another PR, but I guess it is also not really needed.